message_ix/cli.py

```diff
@@ -12,11 +13,13 @@
 from message_ix.default_path_constants import CONFIG_PATH, DEFAULT_MODEL_PATH
 from message_ix.utils import logger
 
+GMS_EXT = ['.gms', '.opt', '.md']
+
 
 def recursive_copy(src, dst, overwrite=False):
     """Copy src to dst recursively"""
     for root, dirs, files in os.walk(src):
-        for f in files:
+        for f in [_f for _f in files if splitext(_f)[1] in GMS_EXT]:
```

Comment:
here I would prefer adding a new kwarg to the function like `skip_ext` and pass `GAMS_EXT` on the function call

Reply:
and if we want to follow @khaeru's lead, we might want to start using the `Path` library, like

```
pathlib.Path(f).suffix
```

Reply:
(same below)

Reply:
done